### ISSUE TITLE
Data Version Field

### ISSUE THREAD

**Issue by krischer** [iris-edu/mseed3-evaluation#12]
Discussion branched off #2. Concerns DRAFT20170622.

@crotwell 

> Field 8 Maybe reserve values < 10 for raw data and qc types of things and values >= 10 for user modified data. The dividing line is whether the metadata still applies, so below 10, the response is still the response. But once the version is above 10, be careful as the response may have already been applied or the data modified to the extent that it no longer can be. In other words, below 10 users can proceed normally, above 10 "here be dragons" and you better know the history. Is 10 large enough?

**Comment by chad-iris** [comment]
> But don't isn't unknown a special case always? Shouldn't "user modified" be > "raw"? Not sure I understand, if I download the raw data, apply the response and save it, we said I was supposed to set the version the "unknown" value, but the data is later than the original data?

For me it's a data center thing, which is good for user <-> data center coordination but otherwise not for users beyond informational.  If users do use it, and how would we prevent them, they need to keep track of their versions and what they mean.  I would recommend that users write extra header(s) to keep track of some processing steps.

**Comment by crotwell** [comment]
@chad-iris So we should drop the "user sets to unknown value on write"? If it is a data center thing, and the end user is not supposed to use it, then they should not change it, ever. So even after processing steps, it stays the same. The meaning is then is not "data version" but "the QC level of the data at the time it was retrieved from the datacenter".

This also allows a user, even after lots of processing steps, to see if the datacenter has issued a new version of the same data and to choose to reprocess in that case.

If that is what you mean, then 👍 from me.

**Comment by chad-iris** [comment]
@crotwell That's mostly what I mean, exception is:
>"the QC level of the data at the time it was retrieved from the datacenter"

I'm very hesitant to attach much notion of quality, unless the definition of quality is "last is best" with no deeper meaning.

**Comment by crotwell** [comment]
@chad-iris Right, my typo, meaning is "the data version at the time is was retrieved". Larger is better, meaning is up to the individual data center. 👍

**Comment by crotwell** [comment]
We might need a better name for this, "data version" makes it sound like this is the current version of the data, which is what I was trying say it is not.

Maybe something like "publication version"?
